### ISSUE TITLE
[Suggestion] Add in-game notification for voice join/part activity

### ISSUE THREAD

**Issue by ethosrot** [ErdbeerbaerLP/Discord-Chat-Integration#162]
It could be beneficial to small/medium servers that use voice chat to communicate when users join/leave (a) voice channel(s). Frequently our players will hop in voice and idle in there, and others would join if they knew people were hanging out in the social channels, but don't watch Discord that closely.

I envision something like the following in-game:

> [ethosrot] what's going on over there?
> [coolguy20] working on my new super project
> [coolguy20 has connected to voice channel _project-planning_]
> [ethosrot] oh shoot, good idea
> [ethosrot has connected to voice channel _project-planning_]
> ...
> [ethosrot has disconnected from voice]
> [coolguy20 has disconnected from voice]

Ideally, I could give a list of voice channel IDs to monitor in the config; something like:
```
# Custom Voice Channel ID list to be monitored and announced to in-game chat. Set to 0 to announce voice activity from all channels.
listVoiceChannelIDs = [  "710039193778138135"  ]
```

**Comment by ErdbeerbaerLP** [comment]
Planned as "Addon" now. 

I am currently working on adding an Addon-loader to this mod.
It allows the creation of platform-independant addons. This will be the first addon then.
Thoose addons will be placed into the `/DiscordIntegration-Data/addons/` directory on the server as soon as it gets released.

I actually use this idea as an test-addon right now:
![image](https://user-images.githubusercontent.com/27149563/103576805-00e56900-4ed4-11eb-8624-4e1906a0f2ba.png)

**Comment by ErdbeerbaerLP** [comment]
Version 2.1.0 released for 1.16.4+

Get the Addon here:
https://github.com/ErdbeerbaerLP/DiscordIntegration-Voice-Notifications/releases

**Comment by rwdj** [comment]
https://github.com/ErdbeerbaerLP/DiscordIntegration-Voice-Notifications/issues/1
